code/modules/client/preferences_toggles.dm

```diff
@@ -223,4 +223,16 @@
 	if (prefs.toggles_secondary & PARALLAX_IS_STATIC)
 		to_chat(src, "Space will no longer move.")
 	else
-		to_chat(src, "Space will now move.")
\ No newline at end of file
+		to_chat(src, "Space will now move.")
+
+/client/verb/switch_health_scanner_output()
+	set name = "Switch health scanner output"
+	set category = "Preferences"
+	set desc = "Switches health scanner output between chat and UI."
+
+	prefs.health_scanner_ui_output = !prefs.health_scanner_ui_output
+
+	if (prefs.toggles_secondary)
+		to_chat(src, "Handheld health scanner will now display results in UI window.")
+	else
+		to_chat(src, "Handheld health scanner will now display results in the chat window.")
```

Comment:
You are using the wrong pref here.
And we do not want to add a toggle for such a minor thing.
Its either one or the other.
At most its a device specific toggle.

Reply:
> You are using the wrong pref here.
> And we do not want to add a toggle for such a minor thing.
> Its either one or the other.
> At most its a device specific toggle.

This was confusing. Because initially I was making it just like you want. But later Matt wanted it to become the preference. After Matt discussed it with Skull, Skull also wants it to not be a preference. Matt said he does not care either way now. I have reverted it back now to be per device-specific toggle.